src/util/util_log.cpp

```diff
@@ -1,8 +1,26 @@
 #include "util_log.h"
 #include "util_env.h"
+#include "util_string.h"
+
+using PFN_wineDbgOutput = int(__cdecl*)(const char*);
+
+static PFN_wineDbgOutput wineDbgOutput = nullptr;
 
 namespace dxvk::log {
+    void cout(const std::string& message) {
+        if (wineDbgOutput)
+            wineDbgOutput((message + '\n').c_str());
```

Comment:
I don't know C++ lifetime rules well enough so I'm not sure if that's necessary but I'd assign `message + '\n'` to a variable just to make sure it doesn't stop existing once we call `c_str()` on it and lose it from the scope.